docs/zh-cn/userguide/multi_machines_deployment.md

```diff
@@ -0,0 +1,111 @@
+# Dragonfly 多机部署
+
+多机部署文档旨在帮助您在多台机器上充分体验 Dragonfly，如需在生产环境使用 Dragonfly 完成生产级别的镜像与文件分发，请参考 supernode 和 dfget 的详细生产级别配置参数。
+
+## 前提条件
+
+我们假设需要用户准备3台机器，一台扮演 supernode 的角色，另外两台扮演 dfclient 的角色，拓扑结构图如下：
+
+![dragonfly cluster topology](../img/dragonfly-topo.png)
+
+因此，您需要准备以下内容：
+
+1. 准备三台在统一局域网的三台机器，我们假设3台机器IP用以下名字代替：
+    * **dfsupernode** 用于部署 Dragonfly 服务端
+    * **dfclient0** 用于部署 Dragonfly 客户端
+    * **dfclient1** 用于部署 Dragonfly 客户端
+2. 以上每台机器上都装有 docker 容器引擎。
+
+## 步骤 1：部署服务端（Supernode） 
+
+在机器 `dfsupernode` 上部署 Dragonfly 服务端（Supernode）：
+
+```bash
+docker run -d --name supernode --restart=always -p 8001:8001 -p 8002:8002 \
+    dragonflyoss/supernode:0.3.0 -Dsupernode.advertiseIp=dfsupernode
```

Comment:
s/dfsupernode/$DFSUPERNODE_IP. WDYT?

Reply:
`dfsupernode` is declared above, it could be ip, hostname, domain, or shell variable.